alacritty.yml

```diff
@@ -413,6 +413,9 @@
 # Mouse bindings are specified as a list of objects, much like the key
 # bindings further below.
 #
+# To activate mouse bindings when an application inside Alacritty captures the mouse, the `Shift`
+# modifier is automatically added as requirement.
```

Comment:
I might rephrase this slightly to:

"To trigger mouse bindings when an application running within Alacritty captures the mouse, the `Shift` modifier is automatically added as a requirement."

Additionally, do you think it's worth explaining a possible conflated binding here? For example, this config, in mouse mode, will trigger both actions at the same time.

```yaml
mouse_bindings:
  - { mouse: Middle, action: ToggleFullscreen }
  - { mouse: Middle, mods: Shift, action: IncreaseFontSize }
```

Reply:
I've applied your suggestion, that is indeed a bit nicer and uses the terminology we use in other places too, which is great.

My hope would be that if someone doesn't understand the comment correctly and still adds this binding, that they'd notice both are triggered and then realize what this comment is about.

While adding an explicit example would make things more clear, that would significantly bloat the documentation and I think that fundamentally a conflict like this isn't the end of the world. So my goal would be to make sure people understand why it happens, if it does, rather than trying to prevent every single person from ever running into it.

Does that make sense?